cpp/include/gar/reader/arrow_chunk_reader.h

```diff
@@ -84,7 +84,9 @@ class VertexPropertyArrowChunkReader {
       chunk_table_.reset();
     }
     if (chunk_index_ >= chunk_num_) {
-      return Status::KeyError();
```

Comment:
For me, it is a little confused to decide when to use `KeyError`, and when to use `Invalid`

Reply:
I think, `KeyError` should use in the scenario that user give a key, we can't find a key in our structure like that, otherwise, use a `Invalid`, so here raise `Invalid` is better. Thanks for question, I will fix it

Reply:
> I think, `KeyError` should use in the scenario that user give a key, we can't find a key in our structure like that, otherwise, use a `Invalid`, so here raise `Invalid` is better. Thanks for question, I will fix it

Then, it is not very clear that which can be called as a "Key": internal vertex id? chunk index? property name? property group? vertex/edge label? adj_list type? ....